tests/wpt/metadata/custom-elements/CustomElementRegistry.html.ini

```diff
@@ -1,50 +1,8 @@
 [CustomElementRegistry.html]
   type: testharness
-  [CustomElementRegistry interface must have define as a method]
-    expected: FAIL
-
-  [customElements.define must throw when the element interface is not a constructor]
-    expected: FAIL
-
-  [customElements.define must not throw the constructor is HTMLElement]
-    expected: FAIL
-
-  [customElements.define must throw with an invalid name]
-    expected: FAIL
-
-  [customElements.define must throw when there is already a custom element of the same name]
-    expected: FAIL
-
-  [customElements.define must throw a NotSupportedError when there is already a custom element with the same class]
-    expected: FAIL
-
-  [customElements.define must throw a NotSupportedError when element definition is running flag is set]
-    expected: FAIL
-
-  [customElements.define must check IsConstructor on the constructor before checking the element definition is running flag]
-    expected: FAIL
-
-  [customElements.define must validate the custom element name before checking the element definition is running flag]
-    expected: FAIL
-
-  [customElements.define unset the element definition is running flag before upgrading custom elements]
-    expected: FAIL
-
   [customElements.define must not throw when defining another custom element in a different global object during Get(constructor, "prototype")]
```

Comment:
The test relies on `createElement` creating a custom element which isn't implemented yet:
```
assert_true(iframe.contentDocument.createElement('another-custom-element') instanceof InnerCustomElement);
```